iGCS/Models/Craft/GCSCraftArduPlane.m

```diff
@@ -26,12 +32,31 @@ @implementation GCSCraftArduPlane
         _guidedMode  = APMPlaneGuided;
         _setModeBeforeGuidedItems = NO;
         _icon = [UIImage imageNamed:@"plane-icon.png"];
+        _customModeNames = @{@(APMPlaneManual): @"Manual",
+                             @(APMPlaneCircle): @"Circle",
+                             @(APMPlaneStabilize): @"Stabilize",
+                             @(APMPlaneFlyByWireA): @"FBW_A",
+                             @(APMPlaneFlyByWireB): @"FBW_B",
+                             @(APMPlaneFlyByWireC): @"FBW_C",
```

Comment:
Have you listened to all of these? Wondering how they sound, and whether we might need a different representation for text and speech?